libs/RobotKit/include/RobotController.h

```diff
@@ -82,13 +82,13 @@ class RobotController : public interface::RobotController
 		rtos::ThisThread::sleep_for(3s);
 	}
 
-	void startSleepTimeout() final
+	void startSleepTimeout(int duration) final
```

Comment:
let use `<chrono>` here

```suggestion
	void startSleepTimeout(std::chrono::seconds duration) final
```